### PR TITLE
CLDR-17995 Clean up site content

### PR DESCRIPTION
**Pull request by macchiati**
CLDR-17995

See comments for some undone steps. Otherwise the items in CLDR-17995 should be done.
The [sitemap](https://5b3030a8.cldr.pages.dev/sitemap) should be helpful for checking site movements.

- [ ] This PR completes the ticket.

ALLOW_MANY_COMMITS=true

### CODE REVIEW

File: docs/site/sitemap.tsv [hunk 1]
```diff
@@ -76,80 +75,82 @@ index
 			translation/translation-guide-general/references
 			translation/unique-translations
 			translation/getting-started/fixing-errors
-		index/survey-tool
+		index/survey-tool
```
Comment (btangmu) [inline docs/site/sitemap.tsv:78]: some added white space at end of line here?

Comment (macchiati) [reply docs/site/sitemap.tsv:78]: That is deliberate; see the notes at the top. It allows github to present in tabular form.

**Comment by macchiati** [comment]
Could someone please review this? I don't want files to drift underneath it.

**Review by DavidLRowe — APPROVED** [review]
LGTM

**Comment by macchiati** [comment]
Thanks!

**Pull request merged**